.eslintignore

```diff
@@ -1,3 +1,7 @@
 /src/testData
 /src/templates
-/src/babelPlugin/test.compiled.carmi.js
\ No newline at end of file
+/src/babelPlugin/test.compiled.carmi.js
+/website/
+/src/tmp.js
```

Comment:
pretty sure this file and out.js arent generated but were forgot there so should be deleted...